frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -830,15 +932,12 @@ export function Experiment_(): JSX.Element {
                                     }
                                 </Row>
                             </>
-                        ) : experimentResultsLoading ? (
-                            <div className="text-center">
-                                <Spinner />
-                            </div>
                         ) : (
-                            <span style={{ fontWeight: 500 }}>
-                                There are no results for this experiment yet.{' '}
-                                {!experimentData.start_date && 'Launch this experiment to start it!'}
-                            </span>
+                            experimentResultsLoading && (
```

Comment:
instead of the spinner, default state now seems to be the text "Loading Data...", which isn't great.

Reply:
(try throttling on chrome)

Reply:
This affects non-FF'ed people, so think we should fix this before merging